source/chapters/1/intro.md

```diff
@@ -91,37 +101,50 @@ This is the basic folder structure that an application, such as the one you've j
     └── main.js // this file contains: import /imports/startup/server
 </pre>
 
+This is not the structure of the application you have just created ! 
+You need to create a new structure for the application, in order for your project to be better organized, and, as a 
+result, have a somewhat easier life. To do this, run these commands in your terminal :
+
 ```
-// to create it quickly, run the commands (they should work just fine on Mac/Linux)
 mkdir -p client server imports/ui imports/startup/client imports/startup/server imports/api
 echo "import '/imports/startup/client';" > client/main.js
 echo "import '/imports/startup/server';" > server/main.js
 ```
-
+![Windows](../images/windowsSmall.png "Windows icon")
+On windows, the script syntax is somewhat different. To achieve the same results on Windows as you would with the 
```

Comment:
It should also specify that he needs to be in the root of project, not just running this in random place, or immediately after he opens cmd

Reply:
Done.